Frontend/cypress/integration/Form.spec.js

```diff
@@ -0,0 +1,13 @@
+describe('Visit site', () => {
+  it('run site!!', () => {
+    cy.visit('http://localhost:3000/');
+  });
+  it('Adding a task', () => {
+    cy.get('[data-testid=inputForm]').type('johnny')
```

Comment:
You probably want to check for the string `johnny` to be present after you have clicked the submit button right?